### PR TITLE
Add Camera Capture Section to UI Reference

### PR DESCRIPTION
**Pull request by CommanderFoo**
UI reference now has a Camera Capture section. It's more of a mini tutorial that follows a similar setup to the gatherables so people will find it familiar.

### CODE REVIEW

File: src/references/ui.md [hunk 1]
```diff
@@ -529,6 +529,125 @@ UI.SetCanCursorInteractWithUI(true)
...
+1. In **Core Content** search for `Plane 1m - One Sided` and add it into the **Client Context** and rename it to `Background`.
+2. Set the **Position** in the **Properties** window to **X** `601`, **Z** `350`.
+3. Set the **Rotation** in the **Properties** window to **Y** `90`.
+4. Set the **Scale** in the **Properties** window to **X** `6`, **Y** `6`, **Z** `6`.
+
+#### Create Frame
+
+The frame will show in the rendered image. This is optional.
+
+1. In **Core Content** search for `pipe 4 sided thin` and add it into the **Client Context** and rename it to `Frame`.
```
Comment (DoubleA-MG) [inline src/references/ui.md:591]: Pipe - 4-Sided Thin

not sure if you were lowercasing and removing symbols for easier search

File: src/references/ui.md [hunk 2]
```diff
@@ -529,6 +529,125 @@ UI.SetCanCursorInteractWithUI(true)
...
+| Medium | 16 |
+| Large | 4 |
+| Very Large | 1 |
+
+### Camera Setup
+
+A [Camera](../api/camera.md) will need to be added to the **Hierarchy** that will be setup just for taking captures. The camera will need certain properties changed so it is setup optimally for creating captures. If any of these properties are not changed, Core will display a warning in the **Event Log** with information on which properties to change.
+
+1. Create a **Client Context** in the **Hierarchy**.
+2. Set the **Position** in the **Properties** window to **X** `-300`, **Z** `350`.
```
Comment (DoubleA-MG) [inline src/references/ui.md:551]: I do like this style for setting Vector3 properties, may have to steal that

**Review by DoubleA-MG — COMMENTED** [review]
Overall looks good. In the future, I feel like this may be more suited as a tutorial like making a picture booth or a camera based game (paparazzi for example). Just seems a bit out of place in a reference.

File: src/references/ui.md (hunk 1 above)
Comment (CommanderFoo) [reply src/references/ui.md:591]: Yep, I do a search to see what I can remove from the term that would still just show that item.

Comment (DoubleA-MG) [reply src/references/ui.md:591]: ok makes sense then. just seemed weird because you wrote "Plane 1m - One Sided" for the previous search

**Comment by CommanderFoo** [comment]
@DoubleA-MG Yeah, it is more like mini tutorial to help people that need to get started, so doesn't go into refreshing and releasing.

The goal for now is to have something that people can use until we get the tutorial done. I am actually working on the tutorial project which will cover the basics (similar to what is in this reference), and we will likely pull some of that out of the reference and link to the tutorial.

**Review by DoubleA-MG — APPROVED** [review]

File: src/references/ui.md (hunk 1 above)
Comment (CommanderFoo) [reply src/references/ui.md:591]: Yeah, that was likely a copy and paste. Might change that as well.

**Pull request merged**